charts/spire/charts/spire-agent/templates/configmap.yaml

```diff
@@ -56,15 +56,29 @@ agent:
   {{- end }}
   {{- end }}
 
+{{- $nodeAttestorUsed := add (len .Values.customPlugins.nodeAttestor) (len .Values.unsupportedBuiltInPlugins.nodeAttestor) }}
+{{- $keyManagerUsed := add (len .Values.customPlugins.keyManager) (len .Values.unsupportedBuiltInPlugins.keyManager) }}
```

Comment:
How are we handling this conflict elsewhere?

Reply:
This code was basically copy/pasted from the spire-server where we are already doing it.

Reply:
https://github.com/spiffe/helm-charts-hardened/blob/main/charts/spire/charts/spire-server/templates/configmap.yaml#L109-L150

Reply:
Thanks for clarifying.